git/hooks/pre-push

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+HOOKS_DIR=$(dirname "$0")
+REPO_DIR="${HOOKS_DIR}/../.."
+source "${REPO_DIR}/util/logging.sh"
+source "${REPO_DIR}/util/path.sh"
+WPTD_PATH=${WPTD_PATH:-$(absdir "${REPO_DIR}")}
+
+docker inspect wptd-dev-instance > /dev/null 2>&1
+INSPECT_STATUS="${?}"
+
+DOCKER_INSTANCE_PID=""
+if [ "${INSPECT_STATUS}" != "0" ]; then
+  docker run -t -d --entrypoint /bin/bash \
```

Comment:
Do the logs from running an instance (if not already running) show in the console? If not, worth logging "Starting docker instance" or similar.